doc/src/pair_modify.rst

```diff
@@ -53,47 +52,40 @@ Examples
    pair_modify pair lj/cut compute no
    pair_modify pair tersoff compute/tally no
    pair_modify pair lj/cut/coul/long 1 special lj/coul 0.0 0.0 0.0
+   pair_modify pair lj/cut/coul/long special lj 0.0 0.0 0.5 special coul 0.0 0.0 0.8333333
 
 Description
 """""""""""
 
-Modify the parameters of the currently defined pair style.  Not all
-parameters are relevant to all pair styles.
-
-If used, the *pair* keyword must appear first in the list of keywords.
-It can only be used with the :doc:`hybrid and hybrid/overlay <pair_hybrid>` pair styles.  It means that all the
-following parameters will only be modified for the specified
-sub-style.  If the sub-style is defined multiple times, then an
-additional numeric argument *N* must also be specified, which is a
-number from 1 to M where M is the number of times the sub-style was
-listed in the :doc:`pair_style hybrid <pair_hybrid>` command.  The extra
-number indicates which instance of the sub-style the remaining
-keywords will be applied to.  Note that if the *pair* keyword is not
-used, and the pair style is *hybrid* or *hybrid/overlay*\ , then all the
-specified keywords will be applied to all sub-styles.
+Modify the parameters of the currently defined pair style.  If the
+pair style is :doc:`hybrid or hybrid/overlay <pair_hybrid>`, then the
+specified parameters are by default modified for all the hybrid sub-styles.  
 
-The *special* and *compute/tally* keywords can **only** be used in
-conjunction with the *pair* keyword and must directly follow it.
-*special* allows to override the
-:doc:`special_bonds <special_bonds>` settings for the specified sub-style.
-*compute/tally* allows to disable or enable registering
-:doc:`compute \*/tally <compute_tally>` computes for a given sub-style.
-More details are given below.
+.. note::
+
+  The behavior for hybrid pair styles can be changed by using the *pair*
+  keyword, which allows to select a specific sub-style to apply a setting to.
```

Comment:
which allows selection of a specific sub-style to apply all the remaining keywords to.